resources/home/dnanexus/utils/generate_output.py

```
@@ -0,0 +1,149 @@
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Dataframe of aggregated data
+    minimal_vcf : str
+        Output filename for the minimal VCF
+    """
+    vcf_lines = [
+        f"{chrom}\t{pos}\t.\t{ref}\t{alt}\t.\t.\t."
+        for chrom,pos,ref,alt
+        in zip(df['CHROM'], df['POS'], df['REF'], df['ALT'])
```

Ruff (0.14.0)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)